examples/jsm/loaders/FBXLoader.js

```diff
@@ -2698,32 +2698,35 @@ class AnimationParser {
 
 	}
 
-	generateRotationTrack( modelName, curves, initialValue, preRotation, postRotation, eulerOrder ) {
-
-		if ( curves.x !== undefined ) {
-
-			this.interpolateRotations( curves.x );
-			curves.x.values = curves.x.values.map( MathUtils.degToRad );
-
-		}
-
-		if ( curves.y !== undefined ) {
-
-			this.interpolateRotations( curves.y );
-			curves.y.values = curves.y.values.map( MathUtils.degToRad );
+	generateRotationTrack(
+		modelName,
+		curves,
+		initialValue,
+		preRotation,
+		postRotation,
+		eulerOrder
+	) {
```

Comment:
Do you mind reformatting code like in this instance?

This should make the diff smaller and the PR easier to review. In general, it's best if you stick to the existing code style whenever possible.